h2integrate/core/utilities.py

```diff
@@ -202,6 +202,21 @@ def __attrs_post_init__(self):
                 )
 
 
+@define(kw_only=True)
+class CacheBaseConfig(BaseConfig):
```

Comment:
Do we have a place for base config files? I don't think utilities is the best place for it. Maybe we can have a `config_baseclasses` file like the `model_baseclasses` file?